package.json

```diff
@@ -1,24 +1,24 @@
 {
-  "name": "lighthouse-batch",
-  "version": "6.0.0",
-  "description": "Generate lighthouse reports for one or more websites",
-  "main": "index.js",
-  "bin": {
-    "lighthouse-batch": "run.js"
-  },
-  "engines": {
-    "node": ">=10.13"
-  },
-  "author": "Massive",
-  "license": "MIT",
-  "homepage": "https://github.com/mikestead/lighthouse-batch",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mikestead/lighthouse-batch.git"
-  },
-  "dependencies": {
-    "commander": "^2.9.0",
-    "lighthouse": "^6.0.0",
-    "shelljs": "^0.8.4"
-  }
+    "name": "lighthouse-batch",
+    "version": "6.4.1",
```

Comment:
looks like you changed the whitespace in the file?

Also you boosted the lib version to align with lighthouse version, but this lib is independent in its versioning. We just happen to track the same major version as it's nice and easy to understand, and we inherit the breaking changes from lighthouse.

If you could leave the versioning of this lib and just update the dependency version of lighthouse. I can do the lib versioning when I release next.

Reply:
Hey, it's not actually a major version of Lighthouse, it doesn't make any breaking change, I had just taken your text from the last lighthouse upgrade commit by wanting to remain coherent 😄.

I'm going to restore the version of the library as well as the whitespaces that have been modified at save with my IDE.